utils/libtokencap/make_dict_v2.sh

```diff
@@ -0,0 +1,49 @@
+#default values
+timeout_sec=5
+LD_PRELOAD_PATH="/home/${USER}/AFLplusplus/utils/libtokencap/libtokencap.so"
```

Comment:
this should also be a command line parameter, and not required (if could set LD:PRELOAD themselves)